### ISSUE TITLE
Node Development Test Environment

### ISSUE THREAD

**Issue by EricoZhao** [comfyanonymous/ComfyUI#4651]
### Your question

I'm a nodepack developer who programming not good.

At every time development, I always write the code first, and then run it after opening ComfyUI to connect the required conditions to see the results.  It's too slowly.

Is there some tools can do it relaxed(test my code)?

My idea: by jupyter notebook use ComfyUI engine to build one test environment for developers.

**Comment by BrechtCorbeel** [comment]
Yes I've asked this before too, I forgot to do it but it should be possible to create nodes that can be updated and relaunched, but to get those nodes into actual comfy is a security issue, because anyone that implements them onto your system can create nodes and programs into your program while you are running the server.
